miqa/core/rest/email.py

```diff
@@ -0,0 +1,45 @@
+from email.mime.image import MIMEImage
+import mimetypes
+import re
+
+from django.core.mail import EmailMultiAlternatives
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class EmailView(APIView):
+    def post(self, request, format=None):
+
+        msg = EmailMultiAlternatives(
+            request.data['subject'],
+            request.data['body'],
+            'example@kitware.com',  # TODO: replace
```

Comment:
We can use the django setting `DEFAULT_FROM_EMAIL` for this. https://github.com/girder/django-composed-configuration/blob/master/composed_configuration/_email.py#L42